spacy/gold/batchers.py

```diff
@@ -0,0 +1,168 @@
+from typing import Union, Iterator, Iterable, Sequence, TypeVar, List, Callable
+from typing import Optional, Any
+from functools import partial
+import itertools
+
+from .example import Example
+from ..util import registry, minibatch
+
+
+Sizing = Union[Iterable[int], int]
+ItemT = TypeVar("ItemT")
```

Comment:
So are we officially using `TypeVar`s in spaCy now? 😛

Reply:
It just felt right here!